fl4health/clients/nnunet_client.py

```diff
@@ -15,6 +15,7 @@
 from flwr.common.logger import FLOWER_LOGGER, console_handler, log
 from flwr.common.typing import Config, Scalar
 from torch import nn
+from torch.cuda.amp import GradScaler
```

Comment:
Curious on your thoughts on whether mixed precision training should be specific to nnunet, or whether its a feature that should be extended to Basic Client (or maybe it warrants a new client subclass and the nnunet client either subclasses or mixes in the new subclass).

Reply:
I will add it in the backlog as a potential addition that the user can optionally specify